packages/sdk/src/methods/getSolverCapacity.ts

```diff
@@ -0,0 +1,42 @@
+import type { paths } from '../types/index.js'
+import { axios } from '../utils/axios.js'
+import { getClient } from '../client.js'
+import { zeroAddress } from 'viem'
+
+export type GetConfigQueryParams = Required<
+  Pick<
+    NonNullable<paths['/config']['get']['parameters']['query']>,
+    'originChainId' | 'destinationChainId'
+  >
+> &
+  Omit<
+    NonNullable<paths['/config']['get']['parameters']['query']>,
+    'originChainId' | 'destinationChainId'
+  >
+
+export type GetConfigResponse =
+  paths['/config']['get']['responses']['200']['content']['application/json']
+
+export async function getSolverCapacity(
+  data: GetConfigQueryParams
```

Comment:
Would also be nice to pass in axios configuration as an optional parameter

Reply:
Done ✅